pom.xml

```diff
@@ -335,6 +335,16 @@
         <enabled>true</enabled>
       </snapshots>
     </pluginRepository>
+    <!-- configure the EBR plug-in repo -->
+    <pluginRepository>
+      <id>ebr</id>
+      <url>https://repo.eclipse.org/content/repositories/ebr/</url>
```

Comment:
On 04/10/2015 09:24 AM, Max Rydahl Andersen wrote:
> i'm especially interested in why not use 
> http://projects.reficio.org/p2-maven-plugin/1.1.0/manual.html which 
> seem to do the exact same thing but not have the downside of having to 
> run multiple times ?
Those are indeed very similar. I'll ask on the ebr-dev how they compare.
The p2-maven-plugin does have the same issue: output of p2-maven-plugin 
cannot be immediately consumed by Tycho, to aggregate them in a site 
(the p2-maven-plugin directly generates the site, but not through 
feature.xml and/or category.xml).
FYI, there are plans to move Orbit fully to EBR, and EBR was mainly 
designed with that objective. So I prefer consuming something that we 
are almost sure is sustainable than some Maven plugin "in the wild".

Overall, I have the strong impression that EBR is meant to integrate 
more transparently in a Tycho build, whereas p2-maven-plugin is more 
about "just" generating a p2 repo.